src/agat/agat_convertspgff2gtf/config.vsh.yaml

```diff
@@ -0,0 +1,82 @@
+name: agat_convertspgff2gtf
```

Comment:
Can we rename this component to `agat_convert_sp_gff2gtf`? If you do, don't forget to rename the directory as well.